packages/styles/scss/patterns/sections/card-section/_card-section.scss

```diff
@@ -88,6 +88,16 @@
       @include themed-items;
     }
   }
+  .#{$prefix}--card-section.#{$prefix}--card-section--g90 {
+    @include carbon--theme($carbon--theme--g90) {
+      @include themed-items;
+    }
+  }
+  .#{$prefix}--card-section.#{$prefix}--card-section--g100 {
+    @include carbon--theme($carbon--theme--g100) {
+      @include themed-items;
+    }
+  }
```

Comment:
Can you align with @annawen1 on the proper way to do themes. Do we theme it in the story, or do we theme it in the component?

Reply:
@photodow we do have a theme object in Story and passing as prop to the component and in component itself handling the themes right. In this case `CardSectionImages` using the `CardSection` and the `CardSection is using the Card`. And CardSection component has the `_setTheme` function and `CardSection.scss `has the themed rules.

Reply:
so we set the theme through the story with leadspace before, but WS team wanted to have it set in the component itself so that they could change the theme. For the `card` component, I didn't want to have it set in the component itself since other pattern, section, and block components use it and the theme can be set in those components.

Reply:
Hmm... ok, so different use case you're saying here. We need to circle back on the theming across the board I think, but that's for a different day.

Reply:
Ok, @photodow what needs to be done here ?

Reply:
I guess we can leave as is for now

Reply:
Yup I resolved it. This should suffice for now.